### DISCUSSION TITLE
How to delete domain from config?

### DISCUSSION THREAD

**Discussion by mithoska** [docker-mailserver/docker-mailserver#2972] [Q&A] [answered]
Hello,

I am using this image with multiple domains. Setup was easy, no issues with that, but now when some of our domains are not used anymore I just want to remove them from the config. 

I checked setup.sh, but no direction there how to do it. Also read README but it just tells how to add.

Can you please let me know how to delete the unused domains?

TIA!

**Comment by casperklein** [comment]
Not sure if I understand that correctly, but all you have to do is, delete all email addresses/aliases belonging to the domain in question:

`./setup.sh email del someuser@old-domain.tld`
`./setup.sh alias del somealias@some-domain.tld someuser@old-domain.tld`

**Comment by mithoska** [comment]
It's almost an answer to my question, this is part of it. But what happens with /etc/postfix/vhost where the main.cf gets $virtual_mailbox_domains?

Also I use OpenDKIM, what happens with the Keys? Will it be removed either?

**Comment by casperklein** [comment, marked as answer]
> But what happens with /etc/postfix/vhost where the main.cf gets $virtual_mailbox_domains?

The file gets automatically updated (via change-detector service), after removing an alias via `setup.sh`. Alternatively you can recreate the container.

> Also I use OpenDKIM, what happens with the Keys? Will it be removed either?

Nope, but you can delete the keys manually in `./docker-data/dms/config/opendkim/keys/<DOMAIN>`
